### DISCUSSION TITLE
Wasm requests in production not working

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#13653] [Q&A] [answered]
_[From Discord](https://discord.com/channels/372137812037730304/550416151172087808/1152145846352228404)_: 

Hi everyone. Wonder if anyone has any ideas. Our WASM project works fine, locally. makes the necessary API calls. When published to production, it appears the API calls are not working. Tried checking if we could see any request in DEVTools and Fiddler, but we don't see any API traffic from the WASM project running in CHROME. Any ideas what we should look at or how we could see these failing requests?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the request.

There may be multiple reasons for this, but you may have exceptions that are not properly reported in your app, make sure to log them in the browser console or in your app.

**Reply by rizaanlakay** [reply]
We had to add refit to the linker. Then it worked in release mode.
